core/src/test/java/org/elasticsearch/search/sort/SortParserTests.java

```diff
@@ -47,7 +47,7 @@ public void testGeoDistanceSortParserManyPointsNoException() throws Exception {
         sortBuilder.endArray();
         sortBuilder.field("order", "desc");
         sortBuilder.field("unit", "km");
-        sortBuilder.field("sort_mode", "max");
+        sortBuilder.field("mode", "max");
```

Comment:
/me tries to figure out whether this is good. I think it is, cause that means that in the context we will have the right thing depending on settings, while tests use strict mode, while before strict became empty by mistake?